pokemon-checker/src/DataTransferObjects/PokemonDTO.ts

```diff
@@ -30,6 +42,26 @@ class PokemonDTO {
       : null;
     this.frontDefault = pokemonConstructorOptions.sprites.front_default;
     this.frontShiny = pokemonConstructorOptions.sprites.front_shiny;
+
+    //TODO: there has to be a more structured method, this relies on the consistency of the API to map.
```

Comment:
I think we can scooch by with this implementation if we handle it by acknowledging the indices via a client-side enum or interface. Something like:

```
enum pokemonStatIndex = {
  HP = 0,
  ATK = 1,
  DEF = 2,
  SPA = 3,
  SPD = 4,
  SPE = 5
}
```

Or we could even rely on Typescript's [default behaviour](https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums) for enums:

```
enum pokemonStatIndex = {
  HP,
  ATK,
  DEF,
  SPA,
  SPD,
  SPE 
}
```

This is by no means perfect, but interfacing with an API that doesn't conform to our expectations is normal in the wild.